modules/pubmaticBidAdapter.js

```diff
@@ -251,6 +262,14 @@ function _createImpressionObject(bid, conf) {
       h: bid.params.height,
       topframe: utils.inIframe() ? 0 : 1,
     }
+    if (utils.isArray(sizes) && sizes.length > 1) {
+      sizes = sizes.splice(1);
```

Comment:
Hi @pm-manasi-moghe 

I ran the updated unit test and the issue appeared to be the format object was getting an empty array.

I did some further debugging/experimenting and I believe the problem was due to this line of code.

When I added a proper range value as the 2nd param in the `splice` (like the following), the unit test completed successfully in Safari.  I wasn't able to find anything definitive but it seemed like this was needed for the es5 shim of the splice to work.  Can you implement this change?

```
var range = sizes.length - 1;
sizes = sizes.splice(1, range);
```